### PR TITLE
  Disable regex usage since cmdlinetester does not correctly process it

### PR DESCRIPTION
**Pull request by babsingh**
Related: https://github.com/eclipse-openj9/openj9/issues/15235

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

**Comment by babsingh** [comment]
fyi @tajila @pshipton

File: test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml [hunk 1]
```diff
@@ -352,15 +352,25 @@
   <output regex="no" type="required">&lt;/verbosegc&gt;</output>
   <output regex="no" type="success">All allocated blocks were freed.</output>
   <!-- allow memory leaks since the JIT leaks, unfortunately -->
+  <!-- Temporarily disable the usage of regex until the cmdlinetester is fixed.
+  Related: https://github.com/eclipse-openj9/openj9/issues/15235
   <output regex="yes" type="success">[0-9]* allocated blocks totaling [0-9]* bytes were not freed before shutdown!</output>
+  -->
+  <output regex="no" type="success">allocated blocks totaling</output>
```
Comment (pshipton) [inline test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml:359]: We should leave this part out, it seems a little more generic. Any success condition will allow the test to pass.

**Comment by pshipton** [comment]
I doubt this is temporary. It doesn't seem worth the effort to come back and restore this later, even it does somehow get fixed. Using `bytes were not freed before shutdown!` seems a perfectly adequate success string.

File: test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml (hunk 1 above)
Comment (babsingh) [reply test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml]: Updated.

**Comment by pshipton** [comment]
Pls fix the copyright.

**Review by pshipton — APPROVED** [review]

**Pull request merged**